source/install/install-ubuntu-1804-mattermost.rst

```diff
@@ -0,0 +1,91 @@
+Installing Mattermost Server
+----------------------------
+
+Install Mattermost Server on a 64-bit machine.
+
+Assume that the IP address of this server is 10.10.10.2.
+
+**To install Mattermost Server on Ubuntu**
+
+1. Log in to the server that will host Mattermost Server and open a terminal window.
+2. Download `the latest version of the Mattermost Server <https://mattermost.com/download/>`__. In the following command, replace ``X.X.X`` with the version that you want to download:
+  ``wget https://releases.mattermost.com/X.X.X/mattermost-X.X.X-linux-amd64.tar.gz``
+3. Extract the Mattermost Server files.
+  ``tar -xvzf mattermost*.gz``
+4. Move the extracted file to the ``/opt`` directory.
+  ``sudo mv mattermost /opt``
+5. Create the storage directory for files.
+  ``sudo mkdir /opt/mattermost/data``
+  .. note::
+    The storage directory will contain all the files and images that your users post to Mattermost, so you need to make sure that the drive is large enough to hold the anticipated number of uploaded files and images.
+6. Set up a system user and group called ``mattermost`` that will run this service, and set the ownership and permissions.
+  a. Create the Mattermost user and group:
+    ``sudo useradd --system --user-group mattermost``
+  b. Set the user and group *mattermost* as the owner of the Mattermost files:
+    ``sudo chown -R mattermost:mattermost /opt/mattermost``
+  c. Give write permissions to the *mattermost* group:
+    ``sudo chmod -R g+w /opt/mattermost``
+7. Set up the database driver in the file ``/opt/mattermost/config/config.json``. Open the file in a text editor and make the following changes:
+  -  If you are using PostgreSQL:
+    1.  Set ``"DriverName"`` to ``"postgres"``
+    2.  Set ``"DataSource"`` to the following value, replacing ``<mmuser-password>``  and ``<host-name-or-IP>`` with the appropriate values:
+     ``"postgres://mmuser:<mmuser-password>@<host-name-or-IP>:5432/mattermost?sslmode=disable&connect_timeout=10"``.
+  -  If you are using MySQL:
+    1.  Set ``"DriverName"`` to ``"mysql"``
+    2.  Set ``"DataSource"`` to the following value, replacing ``<mmuser-password>``  and ``<host-name-or-IP>`` with the appropriate values. Also make sure that the database name is ``mattermost`` instead of ``mattermost_test``:
```

Comment:
```suggestion
    b.  Set ``"DataSource"`` to the following value, replacing ``<mmuser-password>``  and ``<host-name-or-IP>`` with the appropriate values. Also make sure that the database name is ``mattermost`` instead of ``mattermost_test``:
```